must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -30,10 +30,8 @@ ceph_commands+=("ceph mon dump")
 ceph_commands+=("ceph df")
 ceph_commands+=("ceph report")
 ceph_commands+=("ceph osd df tree")
-ceph_commands+=("ceph fs dump --format json-pretty")
```

Comment:
This command `ceph fs dump` and also `ceph health detail` below are removed completely from the list of commands. Are we not losing any piece of information here? (I.e. is the same information in the json output of other commands?)

Reply:
added `ceph fs dump` back and `ceph health detail` is already present

Reply:
we are now collecting both json and standard output of the ceph commands.